### ISSUE TITLE
Unhandled exception. System.DllNotFoundException: Unable to load shared library 'MonoPosixHelper' or one of its 

### ISSUE THREAD

**Issue by rcollette** [Daniel15/prometheus-net.SystemMetrics#6]
With version 2.0.0 of this package

I am getting the following error on application start when using the docker image `mcr.microsoft.com/dotnet/aspnet:7.0.0-alpine3.16`

I am starting using the basic configuration (no customization) `services.AddSystemMetrics();`

```bash
Unhandled exception. System.DllNotFoundException: Unable to load shared library 'MonoPosixHelper' or one of its dependencies. In order to help diagnose loading problems, consider using a tool like strace. If you're using glibc, consider setting the LD_DEBUG environment variable: 
Error loading shared library /app/runtimes/linux/native/MonoPosixHelper.so: No such file or directory
Error loading shared library /app/runtimes/linux-arm64/native/MonoPosixHelper.so: No such file or directory
Error loading shared library /usr/share/dotnet/shared/Microsoft.NETCore.App/7.0.0/MonoPosixHelper.so: No such file or directory
Error loading shared library /app/runtimes/linux-arm64/lib/netstandard2.0/MonoPosixHelper.so: No such file or directory
Error loading shared library MonoPosixHelper.so: No such file or directory
Error loading shared library /app/runtimes/linux/native/libMonoPosixHelper.so: No such file or directory
Error loading shared library ld-linux-aarch64.so.1: No such file or directory (needed by /app/runtimes/linux-arm64/native/libMonoPosixHelper.so)
Error loading shared library /usr/share/dotnet/shared/Microsoft.NETCore.App/7.0.0/libMonoPosixHelper.so: No such file or directory
Error loading shared library /app/runtimes/linux-arm64/lib/netstandard2.0/libMonoPosixHelper.so: No such file or directory
Error loading shared library libMonoPosixHelper.so: No such file or directory
Error loading shared library /app/runtimes/linux/native/MonoPosixHelper: No such file or directory
Error loading shared library /app/runtimes/linux-arm64/native/MonoPosixHelper: No such file or directory
Error loading shared library /usr/share/dotnet/shared/Microsoft.NETCore.App/7.0.0/MonoPosixHelper: No such file or directory
Error loading shared library /app/runtimes/linux-arm64/lib/netstandard2.0/MonoPosixHelper: No such file or directory
Error loading shared library MonoPosixHelper: No such file or directory
Error loading shared library /app/runtimes/linux/native/libMonoPosixHelper: No such file or directory
Error loading shared library /app/runtimes/linux-arm64/native/libMonoPosixHelper: No such file or directory
Error loading shared library /usr/share/dotnet/shared/Microsoft.NETCore.App/7.0.0/libMonoPosixHelper: No such file or directory
Error loading shared library /app/runtimes/linux-arm64/lib/netstandard2.0/libMonoPosixHelper: No such file or directory
Error loading shared library libMonoPosixHelper: No such file or directory

   at Mono.Unix.Native.Syscall.sysconf(SysconfName name, Errno defaultError)
   at Mono.Unix.Native.Syscall.sysconf(SysconfName name)
   at Prometheus.SystemMetrics.Collectors.LinuxCpuUsageCollector.CreateMetrics(MetricFactory factory)
   at Prometheus.SystemMetrics.SystemMetricsHostedService.StartAsync(CancellationToken cancellationToken)
   at Microsoft.Extensions.Hosting.Internal.Host.StartAsync(CancellationToken cancellationToken)
   at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.RunAsync(IHost host, CancellationToken token)
   at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.RunAsync(IHost host, CancellationToken token)
   at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.Run(IHost host)
  
```

**Comment by Daniel15** [comment]
Unfortunately the Mono.Posix doesn't have an ARM64 build (see https://github.com/mono/mono.posix). I'll see if I can remove that dependency.

**Comment by Daniel15** [comment]
I just published v3.1.0 which removes the dependency on `Mono.Posix`. Could you please try it out and let me know if it works for you? I don't have any ARM64 systems to test with.
